src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,9 +576,9 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
-    
+
     // ===================== Utility methods used by some child classes========
-    
+
```

Comment:
Just a quick advice that "everything looks the same" based on human eyesight can be dangerous when computer files are involved. There are things such as trailing whitespaces, different line endings, different encodings, etc. For example, these two are *not* the same:
```
	Some text
```
```
        Some text
```